docs/pipelines/access-control/index.md

```diff
@@ -0,0 +1,29 @@
+# Access control
+
+Gruntwork Pipelines grants permissions by defining three [GitHub Teams](https://docs.github.com/en/organizations/organizing-members-into-teams/about-teams), which should map to three separate personas in your organization. Each team and its permissions are designed to grant the _least possible permissions_ to each individual (or machine user) in your organization for them to be able to perform changes to your infrastructure.
+
+### GitHub Team: infrastructure-administrators
```

Comment:
I'm not sure these need to be subsections; should we just use a list to enumerate the 3-team structure?

Are we explicitly recommending these string literals as team slugs? If so, the lowercase-hyphenated form makes sense, but they should be rendered as inline code. If not, I think we should just recast this as e.g. "Infrastructure administrators GitHub team"

Reply:
Yeah, that is the recommendation. It doesn't matter from pipelines perspective, it just seems like a reasonable approach for naming. Can update formatting and remove the subsections.

Reply:
Formatting is updated now